packages/schemas/tables/applications.sql

```diff
@@ -6,8 +6,7 @@ create table applications (
   description text,
   type application_type not null,
   oidc_client_metadata jsonb /* @use OidcClientMetadata */ not null,
-  id_token_ttl bigint not null default(86400),
-  refresh_token_ttl bigint not null default(2592000),
+  custom_client_metadata jsonb /* @use CustomClientMetadata */,
```

Comment:
jsonb should be not null in general? we can give a `{}` instead

Reply:
updated